pkg/executor/cte.go

```diff
@@ -762,8 +761,8 @@ func (p *cteProducer) checkAndUpdateCorColHashCode() bool {
 	return changed
 }
 
-func (p *cteProducer) logTbls(ctx context.Context, err error, iterNum uint64) {
```

Comment:
why modify this?

Reply:
info log when failpoint fail to help debug problems